apis/memcached/v1beta1/memcached_types.go

```diff
@@ -45,6 +45,12 @@ type MemcachedSpec struct {
 type MemcachedStatus struct {
 	// Conditions
 	Conditions condition.Conditions `json:"conditions,omitempty" optional:"true"`
+
+	// ServerList - List of memcached endpoints without inet(6) prefix
+	ServerList []string `json:"serverList,omitempty" optional:"true"`
+
+	// ServerListWithPrefix - List of memcached endpoints with inet(6) prefix
+	ServerListWithPrefix []string `json:"serverListWithInet,omitempty" optional:"true"`
```

Comment:
* Out of curiosity: Why we need to have two lists?
* The golang struct name `ServerListWithPrefix` could be synced with the name of the json field. (I prefer the json field name)

Reply:
In OpenStack we have two implementations to use memcache. One is based on python-memcached while the other depends on pymemcache. pymemcache is much modern and does not require special treatment for IPv6, but python-memcached requires inet6 prefix in case the service uses IPv6 address.

We may probably just expose prefix and let clients compose the full server list but for that we also have to expose port. So I decided to compose the list items here in memcached controller.

I agree with the naming preference so I'll fix that.

Reply:
I see. Thanks for the explanation. If we need the prefix for hte clients then I think we can keep the two lists. I would like to ask you to summarize the above reasoning in a code comment so that we will know in the future why we have a the duplication

Reply:
I've added a comment in the logic.